### ISSUE TITLE
knitr::kable is not rendered in the output of a chunk in R 3.5.0

### ISSUE THREAD

**Issue by chenyenchung** [rstudio/rmarkdown#1353]
After upgrading to R 3.5.0 and RStudio 1.1.453, chunk output with `knitr::kable()` is no longer rendered but kept as raw markdown.
- This happens in the output below a chunk and preview of R Notebook:

![2018-05-17 2 36 46](https://user-images.githubusercontent.com/22821107/40160335-d2664d1a-59df-11e8-9233-e680145f7858.png)

- While the result of knit to html is fine.

The output used to be like this in R 3.4.3:
![old_r](https://user-images.githubusercontent.com/22821107/40160251-58c7bcbe-59df-11e8-9551-6fde721dd7aa.png)

It looks like this now:
![r3 5](https://user-images.githubusercontent.com/22821107/40160252-58f08630-59df-11e8-9aaa-d9f1b174c011.png)

On my machine, the minimal example is to open a new Rmarkdown or R Notebook file in RStudio, and run this chunk:

````
```{r}
knitr::kable(iris)
```
````

I also tried `{r results='asis'}` and installed development version of rmakrdown with `devtools::install_github("rstudio/rmarkdown")`, but neither of them fixed the issue.

![asis](https://user-images.githubusercontent.com/22821107/40160408-28438c20-59e0-11e8-9506-b20610a2f4f2.png)

Thank you so much for all the help! If there's any thing else I could test, please let me know. This issue happened on a machine with the session info as below:

```
> sessionInfo()
R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.4

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.16     digest_0.6.15    withr_2.1.2      rprojroot_1.3-2 
 [5] R6_2.2.2         jsonlite_1.5     backports_1.1.2  magrittr_1.5    
 [9] git2r_0.21.0     evaluate_0.10.1  highr_0.6        httr_1.3.1      
[13] stringi_1.2.2    curl_3.2         rmarkdown_1.9.12 devtools_1.13.5 
[17] tools_3.5.0      stringr_1.3.1    yaml_2.1.19      compiler_3.5.0  
[21] base64enc_0.1-3  memoise_1.1.0    htmltools_0.3.6  knitr_1.20  
```

**Comment by yihui** [comment]
Duplicate of #1331

**Comment by chenyenchung** [comment]
Currently in RStudio 1.1.453 the kable remains displayed as raw markdown in my machine. 

![new](https://user-images.githubusercontent.com/22821107/40187488-36046484-5a2a-11e8-9a44-919f0d4b7bc2.png)

I tested on another machine with current version of RStudio and R 3.4.2, the rendering is fine (session info below):

```
# Before upgrade
R version 3.4.2 (2017-09-28)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.4

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base    

loaded via a namespace (and not attached):
[1] compiler_3.4.2  backports_1.1.2 magrittr_1.5    rprojroot_1.3-2 htmltools_0.3.6
[6] tools_3.4.2     yaml_2.1.18     Rcpp_0.12.16    stringi_1.1.7   rmarkdown_1.9  
[11] highr_0.6       knitr_1.20      stringr_1.3.0   digest_0.6.15   evaluate_0.10.1
```

Installing R 3.5.0 breaks the rendering, so I feel this is somewhat reproducible.

```
# After upgrade
R version 3.5.0 (2018-04-23)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.4

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base    

loaded via a namespace (and not attached):
[1] compiler_3.5.0 tools_3.5.0    yaml_2.1.19    highr_0.6      knitr_1.20
```

**Comment by yihui** [comment]
@kevinushey @jmcphers I feel this is a issue similar to #1331, and the fix in the IDE might be similar, too. Could you take a look at it when you have a chance? Thanks!

**Comment by 33Vito** [comment]
I am having the same issue with R 3.5.0 and Rstudio 1.1.453 - kable produces either raw HTML or raw markdown. 

I found a temporary workaround using the `kable_styling` function from the [kableExtra](https://github.com/haozhu233/kableExtra) package. i.e.

*`kable(head(mtcars))` produces raw html
* `kable_styling(kable(head(mtcars)))` produces rendered table in both Rstudio and the associated R notebook. 

Hope this would be fixed soon!

**Comment by jmcphers** [comment]
This is fixed the dailies as well as in RStudio 1.1.455 (not yet released). You can try the latter here:

http://s3.amazonaws.com/rstudio-dailybuilds/RStudio-1.1.455.dmg

**Comment by chenyenchung** [comment]
Thank you so much for the timely fix. I just downloaded the new version of RStudio, and tables are rendered perfectly now.
